sos/report/plugins/openstack_edpm.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2023 Red Hat, Inc., Roberto Alfieri <ralfieri@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class OpenStackEDPM(Plugin, RedHatPlugin):
+
+    short_desc = 'Installation information from OpenStack EDPM deployment'
+
+    plugin_name = 'openstack_edpm'
+    profiles = ('openstack', 'openstack_edpm')
+
```

Comment:
We should have at least one of `packages`, `files`, `commands`, `containers`, or `services` defined here to gate plugin enablement. Otherwise this will be enabled on every `sos report` execution.